main/getting-started/intro-zoe.md

```diff
@@ -257,27 +254,24 @@ be compared to other installations, and you can call
 
 Now Alice uses the installation to create a new instance. She must
 also tell Zoe about the ERTP issuers she wants to use, by specifying
-their role with Keywords. Alice was escrowing moola, so she uses the
-keyword `Asset` to label the `moolaIssuer`. She wanted simoleans, so
+their role with Keywords. Alice was escrowing Moola, so she uses the
+keyword `Asset` to label the `moolaIssuer`. She wanted Simoleans, so
 she uses the keyword `Price` to label the `simoleanIssuer`. When you
-create a new instance of the contract ([see permalink to line
-58](https://github.com/Agoric/agoric-sdk/blob/23c3f9df56940230e21a16b4861f40197192fdea/packages/zoe/src/contracts/atomicSwap.js#L58))
-by calling `startInstance()`, it returns a `creatorInvitation`. Alice
-uses this to make her offer; even the instance's creator needs to have
-an invitation to it to participate in it.
+create a new instance of the contract
+by calling `startInstance()`, it returns a `creatorInvitation`
+(even the instance's creator needs to have
+an invitation to participate in it).
```

Comment:
```suggestion
by calling `startInstance()`, it returns a `creatorInvitation`.
(Even the instance's creator needs to have
an invitation to participate in it.)
```
In this case, the parenthetical is a complete sentence, so I'd end the previous sentence and make it a proper sentence. I think. Don't follow my advice if it violates writing norms.